src/haddock/modules/analysis/caprieval/capri.py

```diff
@@ -671,14 +680,16 @@ def _load_atoms(
             PDB file of the model to have its atoms identified
         full : bool
             If False, only backbone atoms will be retrieved, otherwise all atoms
+        ff : string
+            Force-field information : all-atom, martini2 or martini3
```

Comment:
Because the desired value for all-atom is "aa", I would put "aa" in the docstring.
Also, martini3 is not implemented yet. But I get the point of putting it now already as it might be scheduled for the near future.

This applied for all docstrings describing theis parameter
```suggestion
            Force-field information : aa (all-atom), martini2 or martini3
```